scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/executors/ForkedProcessBuilderCreator.java

```diff
@@ -126,4 +129,19 @@ private OSProcessBuilder getOsProcessBuilder(TaskContext context, File workingDi
         }
         return processBuilder;
     }
+
+    private void addExecutionPermissionToScripts() throws IOException {
+        String schedulerHome = ClasspathUtils.findSchedulerHome();
```

Comment:
Include this instruction in the if block
String schedulerHome = ClasspathUtils.findSchedulerHome();

Reply:
ok